rbac/management/group/view.py

```diff
@@ -97,9 +96,9 @@ def get_serializer_class(self):
             return GroupInputSerializer
         return GroupSerializer
 
-    def protect_default_groups(self, uuid, action):
+    def protect_default_groups(self, action):
         """Deny modifications on platform_default groups."""
-        group = get_object_or_404(Group, uuid=uuid)
+        group = self.get_object()
```

Comment:
Nice! Works like magic!!! I was so curious why it works differently although it uses get_object_or_404 too, and found this, they redefined this get_object_or_404: https://github.com/encode/django-rest-framework/blob/master/rest_framework/generics.py#L7